src/test/java/com/wikia/webdriver/testcases/discussions/PromotingTests.java

```diff
@@ -74,7 +74,7 @@ public void anonUserOnMobileCanClickGooglePlayLink() {
 
   private Promoting findPromoting() {
     String siteId = Utils.excractSiteIdFromWikiName(MercuryWikis.FALLOUT);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "siteId" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1481)